### ISSUE TITLE
Feature idea: Don't blur images marked as decorative in accessibility checker plugins?

### ISSUE THREAD

**Issue by mrwweb** [kerri-hicks/blur_no-alt#11]
## Problem

I'm currently testing this plugin in tandem with [Editoria11y](https://wordpress.org/plugins/editoria11y-accessibility-checker/). It can be frustrating for users to add the `noalt` class but still have the issue flagged by Editoria11y.

## Current Solution

It's possible to set Editoria11y to ignore images with the `noalt` class used by this plugin.

## Proposed Solution

It strikes me as even more user-friendly to flip the script and have this plugin not blur images marked as OK in other plugins.

That would make this tool super complementary with lots of others that people are using, which feels exciting and community oriented.

_I don't know if this is feasible for all plugins_, but I think I see a path to do it with Editoria11y.

Plugins to consider doing this for:

- Editoria11y - Allows dismissing notice of image without alt text (I don't think Editoria11y's predecessor [Sa11y](https://wordpress.org/plugins/sa11y/) allows for this, so it wouldn't be possible to integrate)
- [WP Accessibility](https://wordpress.org/plugins/wp-accessibility/) - Allows marking images as decorative in Classic Editor and Media Library
- [Equalize Digital Accessibility Checker](https://wordpress.org/plugins/accessibility-checker/) - Allows dismissing warnings for no alt text

## Alternative Solution

Another way to solve this automatically would be to adjust the "Ignore" setting to include `noalt` when the plugin is activated, though I get nervous about silently changing a setting without a way to easily opt-out.
